### PR TITLE
pivot features docs initial

### LINKED ISSUE #611: Port Pivot Grid Features Topic

### PR DESCRIPTION
**Pull request by tishko0**
closes #611

### CODE REVIEW

File: doc/en/components/grids/pivot-grid/pivot-grid-features.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+All chips can change their order within their area by drag & drop.
+The chips from `rows`, `column`, `filter`(dimension chips) can be moved from any of those areas to any other and at any place.
+Chips from these areas can not be moved to the `values` area and chips from the `values` area can not be moved to any of the dimension areas.
+
+>[!NOTE]
+>The chips from the Pivot Grid can not be moved to the Pivot Data Selector and items from the Pivot Data Selector can not be moved to the Pivot Grid.
+
+## API References
+* [{ComponentName}Component]({environment:{Platform}ApiUrl}/classes/{ComponentName}component.html)
+* [IgxPivotDataSelectorComponent]({environment:{Platform}ApiUrl}/classes/igxpivotdataselectorcomponent.html)
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/pivot-grid/pivot-grid-features.md:146]: do not hard-code URLs for API link, instead just list API members in backticks, e.g.
``` `{PivotGridName}` ```

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
please add content for WC/Blazor platforms
use {PivotGridTitle} instead of "Pivot Grid"
use {ProductName} instead of "ignite ui for {Platform}"
use CamelCase in secton headers, e.g. ## This Is Section Header
use {ComponentSelector} only in topics that are under the _shared folder
use {PivotGridSelector} in topics that are NOT  under the _shared folder
use {ComponentName} only in topics that are under the _shared folder
use {PivotGridName} in topics that are NOT  under the _shared folder

File: doc/en/components/grids/pivot-grid/pivot-grid-features.md (hunk 1 above)
Comment (tishko0) [reply doc/en/components/grids/pivot-grid/pivot-grid-features.md]: Updated the docs, please review again

Comment (tishko0) [reply doc/en/components/grids/pivot-grid/pivot-grid-features.md]: Also I am not really sure on filteringExpressionTree for dimensions, there is no value parameter exposed, seems kinda incomplete at this point

**Comment by mddragnev** [comment]
@HUSSAR-mtrela Note that if using {PivotGridTitle}, {PivotGridName} and etc. in here: 
```
---
title: {Platform} {ComponentTitle} Features | Pivot Tables | Infragistics
_description: Create fast, responsive {Platform} {ComponentTitle}s and tables with {ProductName}. Perform complex data analysis via pivot data.
_keywords: {Platform} {ComponentTitle}, {Platform} material pivot table, {ProductName}, {ComponentTitle} features, pivot features
mentionedTypes: ['Infragistics.Controls.Grid']
---
```

The docs throw error and does not generate. I dont know if that is an issue but thats why I have changed them to be {ComponentTitle}. Note that if you use {PivotGridTitle} in the sections below there is no problem.

**Review by HUSSAR-mtrela — COMMENTED** [review]
postpone PR until we release pivot grid with just overview topic

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**